tests/e2e/pageObjects/base-run-commands-page.ts

```diff
@@ -0,0 +1,78 @@
+import { Selector, t } from 'testcafe';
+import { InstancePage } from './instance-page';
+
+export class BaseRunCommandsPage extends InstancePage {
+
+    submitCommandButton = Selector('[data-testid=btn-submit]');
+    queryInput = Selector('[data-testid=query-input-container]');
+
+    // History containers
+    queryCardCommand = Selector('[data-testid=query-card-command]');
+    fullScreenButton = Selector('[data-testid=toggle-full-screen]');
+    rawModeBtn = Selector('[data-testid="btn-change-mode"]');
+    queryCardContainer = Selector('[data-testid^=query-card-container]');
+    reRunCommandButton = Selector('[data-testid=re-run-command]');
+    copyBtn = Selector('[data-testid^=copy-btn-]');
+    copyCommand = Selector('[data-testid=copy-command]');
+
+    runButtonToolTip = Selector('[data-testid=run-query-tooltip]');
+    loadedCommand = Selector('[class=euiLoadingContent__singleLine]');
+    runButtonSpinner = Selector('[data-testid=loading-spinner]');
+    commandExecutionDateAndTime = Selector('[data-testid=command-execution-date-time]');
+    executionCommandTime = Selector('[data-testid=command-execution-time-value]');
+    executionCommandIcon = Selector('[data-testid=command-execution-time-icon]');
+    executedCommandTitle = Selector('[data-testid=query-card-tooltip-anchor]', { timeout: 500 });
+    queryResult = Selector('[data-testid=query-common-result]');
+
+    cssQueryCardCommand = '[data-testid=query-card-command]';
+    cssQueryTextResult = '[data-testid=query-cli-result]';
+    cssReRunCommandButton = '[data-testid=re-run-command]';
+    cssDeleteCommandButton = '[data-testid=delete-command]';
+
+    getTutorialLinkLocator = (tutorialName: string): Selector =>
+        Selector(`[@data-testid="data-testid=query-tutorials-link_${tutorialName}"]`);
+
+    /**
+     * Get card container by command
+     * @param command The command
+     */
+    async getCardContainerByCommand(command: string): Promise<Selector> {
+        return this.queryCardCommand.withExactText(command).parent('[data-testid^="query-card-container-"]');
```

Comment:
I know that you just moved this from workbench-page
but lets store '[data-testid^="query-card-container-"]' separately in css selectors above

Reply:
fixed